overrides/app/pages/algolia-product-list/partials/algolia-product-colors.jsx

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
```

Comment:
As I had said previously, we shouldn't put a salesforce copyright on code that is from us.